tests/test_wellbore_marker_frame.py

```diff
@@ -240,35 +186,62 @@ def test_get_interpretation_object(example_model_and_crs):
+    horizon_marker = resqpy.well.WellboreMarker(parent_model = model,
+                                                parent_frame = wellbore_marker_frame,
+                                                marker_index = 0,
+                                                marker_type = 'horizon',
+                                                interpretation_uuid = horizon_interp1_uuid,
+                                                title = 'horizon_marker')
+
+    woc_marker = resqpy.well.WellboreMarker(parent_model = model,
+                                            parent_frame = wellbore_marker_frame,
+                                            marker_index = 1,
+                                            marker_type = 'water oil contact',
+                                            title = 'woc_marker')
+
+    fault_marker = resqpy.well.WellboreMarker(parent_model = model,
+                                              parent_frame = wellbore_marker_frame,
+                                              marker_index = 2,
+                                              marker_type = 'fault',
+                                              interpretation_uuid = fault_interp_1_uuid,
+                                              title = 'fault_marker')
+    marker_list = [horizon_marker, woc_marker, fault_marker]
+    node_mds = np.array([400, 410, 430])
 
-    wellbore_marker_frame_dataframe = pd.DataFrame({
-        'MD': [300, 310, 330],
-        'Type': ['horizon', 'horizon', 'fault'],
-        'Surface': ['marker_horizon_1', 'marker_horizon_2', 'marker_fault_1'],
-        'Interp_Surface': ['horizon_interp_1', 'horizon_interp_2', 'fault_interp_1'],
-        'Well': well_name
-    })
-
-    # Create a wellbore marker frame from a dataframe
-    wellbore_marker_frame = resqpy.well.WellboreMarkerFrame.from_dataframe(parent_model = model,
-                                                                           dataframe = wellbore_marker_frame_dataframe,
-                                                                           trajectory = trajectory)
     # --------- Act ----------
-    # Get an interpretation based on an uuid
-    # TODO: alter test as get_interpretation_obj() method will have breaking changes or be removed in the future
-    found_interpretation = wellbore_marker_frame.get_interpretation_obj(interpretation_uuid = horizon_interp_2_uuid)
+    # Add attributes to the wellbore marker frame from the marker list
+    wellbore_marker_frame.from_marker_list(md_array = node_mds, marker_list = marker_list)
 
     # --------- Assert ----------
-    assert found_interpretation == horizon_interp_2
+    # Verify that attributes have been added correctly
+    assert wellbore_marker_frame.node_count == len(wellbore_marker_frame.node_mds) == len(
+        wellbore_marker_frame.marker_list) == 3
+    np.testing.assert_equal(node_mds, wellbore_marker_frame.node_mds)
```

Comment:
As elsewhere, make sure to use floating point data and compare arrays with assert_array_almost_equal() from numpy.testing.